core/src/main/java/hudson/model/View.java

```diff
@@ -1219,11 +1219,20 @@ public ContextMenu doChildrenContextMenu(StaplerRequest request, StaplerResponse
         return Jenkins.getInstance().<View,ViewDescriptor>getDescriptorList(View.class);
     }
 
+    /**
+     * Returns the {@link ViewDescriptor} instances that can be instantiated for the {@link ViewGroup} in the current
+     * {@link StaplerRequest}. <strong>NOTE: This method must be called from a {@link StaplerRequest}</strong>
+     * @return the list of instantiable {@link ViewDescriptor} instances for the current {@link StaplerRequest}
+     */
```

Comment:
Note: this is the javadoc that applied to the method *before* my changes as the method would have blown up with a NPE if invoked outside of the `newView.jelly` screen of a `ViewGroup` as soon as `AllView.isInstantiable()` was called